src/node_wasi.h

```diff
@@ -0,0 +1,98 @@
+#ifndef SRC_NODE_WASI_H_
+#define SRC_NODE_WASI_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "base_object.h"
+#include "uvwasi.h"
+
+namespace node {
+namespace wasi {
+
+
+class WASI : public BaseObject {
+ public:
+  WASI(Environment* env,
+       v8::Local<v8::Object> object,
+       uvwasi_options_t* options);
+  static void New(const v8::FunctionCallbackInfo<v8::Value>& args);
+  SET_NO_MEMORY_INFO()
```

Comment:
This method should be implemented and at least point to `memory_`, because that’s a strongly referenced variable, plus (maybe as a TODO) it would also be nice if we could ask uvwasi to tell us how much memory it’s using and report that?